src/signals.c

```diff
@@ -45,8 +45,6 @@ void startup_handle_falling_child()
       break;
     }
   }
-
-  signal(SIGCHLD, startup_handle_falling_child);
 }
```

Comment:
Removed the lines since sigaction doesnt revert back to default signal handlers.